docs/content/sips/008-using-oci-registries.md

```diff
@@ -0,0 +1,364 @@
+title = "SIP 008 - Distributing Spin applications using OCI registries"
+template = "main"
+date = "2023-01-04T01:01:01Z"
```

Comment:
Is it worth bumping the date?

Reply:
I do like to have a record of when this was originally written first.